src/services/test-helpers/responses/pallets/fetchStorage789629.json

```
@@ -0,0 +1,171 @@
+      "modifier": "Default",
+      "type": {
+        "Map": {
+          "hasher": "Twox64Concat",
+          "key": "AccountId",
+          "value": "Voting",
+          "linked": false
+        }
+      },
+      "fallback": "0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+      "documentation": " All votes for a particular voter. We store the balance for the number of votes that we\n have recorded. The second item is the total amount of delegations, that will be added.\n\n TWOX-NOTE: SAFE as `AccountId`s are crypto hashes anyway."
+    },
+    {
+      "name": "Locks",
+      "modifier": "Optional",
+      "type": {
+        "Map": {
+          "hasher": "Twox64Concat",
+          "key": "AccountId",
+          "value": "BlockNumber",
+          "linked": false
+        }
+      },
+      "fallback": "0x00",
+      "documentation": " Accounts for which there are locks in action which may be removed at some point in the\n future. The value is the block number at which the lock expires and may be removed.\n\n TWOX-NOTE: OK ― `AccountId` is a secure hash."
+    },
+    {
+      "name": "LastTabledWasExternal",
+      "modifier": "Default",
+      "type": {
+        "Plain": "bool"
+      },
+      "fallback": "0x00",
+      "documentation": " True if the last referendum tabled was submitted externally. False if it was a public\n proposal."
+    },
+    {
+      "name": "NextExternal",
+      "modifier": "Optional",
+      "type": {
+        "Plain": "(Hash,VoteThreshold)"
+      },
+      "fallback": "0x00",
+      "documentation": " The referendum to be tabled whenever it would be valid to table an external proposal.\n This happens when a referendum needs to be tabled and one of two conditions are met:\n - `LastTabledWasExternal` is `false`; or\n - `PublicProps` is empty."
+    },
+    {
+      "name": "Blacklist",
+      "modifier": "Optional",
+      "type": {
+        "Map": {
+          "hasher": "Identity",
+          "key": "Hash",
+          "value": "(BlockNumber,Vec<AccountId>)",
+          "linked": false
+        }
+      },
+      "fallback": "0x00",
+      "documentation": " A record of who vetoed what. Maps proposal hash to a possible existent block number\n (until when it may not be resubmitted) and who vetoed it."
+    },
+    {
+      "name": "Cancellations",
+      "modifier": "Default",
+      "type": {
+        "Map": {
+          "hasher": "Identity",
+          "key": "Hash",
+          "value": "bool",
+          "linked": false
+        }
+      },
+      "fallback": "0x00",
+      "documentation": " Record of all proposals that have been subject to emergency cancellation."
+    },
+    {
+      "name": "StorageVersion",
+      "modifier": "Optional",
+      "type": {
+        "Plain": "Releases"
+      },
+      "fallback": "0x00",
+      "documentation": " Storage version of the pallet.\n\n New networks start with last version."
```

```suggestion
      "documentation": "Storage version of the pallet.\n\n New networks start with last version."
```

Is there a reason for these leading spaces? My OCD tingles a bit... :)

---

The reason must be upstream from us - I am guessing it is how the Rust docs are converted into the metadata. 

@dvdplm do you think we should strop leading/trailing white spaces?

---

Nah, less moving parts is good. I didn't realize we pull this in from the rust docs (and have no idea how that whole thing works tbh). Ignore me. :)